src/Entity/Register.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\commerce_pos\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+
+/**
+ * Defines the Register entity.
+ *
+ * @ConfigEntityType(
+ *   id = "register",
```

Comment:
Entity type IDs need to be prefixed with the module name. "commerce_pos_register".